Sources/Chat/RegistryManager.swift

```diff
@@ -0,0 +1,35 @@
+
+import Foundation
+import WalletConnectUtils
+import WalletConnectKMS
+
+actor RegistryManager {
```

Comment:
What the difference between Manager and Engine naming?

Reply:
I am always scratching my head when giving them a names. in this context it just sounds better to me. Any suggestions?